src/index/Index.h

```diff
@@ -47,6 +47,14 @@ class Index {
     size_t normalAndInternal_() const { return normal_ + internal_; }
   };
 
+  // Store all information about possible search results in one place.
+  struct WordEntityPostings {
+    vector<TextRecordIndex> cids;
+    vector<WordIndex> wids;
+    vector<Id> eids;
+    vector<Score> scores;
```

Comment:
If we have this central struct now,
we can also document, what its members mean.
Also: I think that these vectors all have the same size with the exception that some of them are sometimes empty
is that also correct? If so, it should also be written here.

Reply:
Yes, I think so too.